src/main/java/uk/gov/hmcts/divorce/document/content/MiniApplicationTemplateContent.java

```diff
@@ -0,0 +1,162 @@
+                templateData.put(JOINT_OR_SOLE, SOLE_APPLICATION);
+
+                if (caseData.getDivorceOrDissolution().isDivorce()) {
+                    templateData.put(CONDITIONAL_ORDER_DIVORCE_OR_CIVIL_PARTNERSHIP, CONDITIONAL_ORDER_OF_DIVORCE_FROM);
+                } else {
+                    templateData.put(CONDITIONAL_ORDER_DIVORCE_OR_CIVIL_PARTNERSHIP, DISSOLUTION_OF_THE_CIVIL_PARTNERSHIP_WITH);
+                }
+            } else {
+                templateData.put(JOINT_OR_SOLE, JOINT_APPLICATION);
+
+                if (caseData.getDivorceOrDissolution().isDivorce()) {
+                    templateData.put(CONDITIONAL_ORDER_DIVORCE_OR_CIVIL_PARTNERSHIP, CONDITIONAL_ORDER_OF_DIVORCE);
+                } else {
+                    templateData.put(CONDITIONAL_ORDER_DIVORCE_OR_CIVIL_PARTNERSHIP, DISSOLUTION_OF_A_CIVIL_PARTNERSHIP);
+                }
+            }
+
+            if (caseData.getDivorceOrDissolution().isDivorce()) {
+                templateData.put(DIVORCE_OR_DISSOLUTION, FOR_A_DIVORCE);
+                templateData.put(MARRIAGE_OR_RELATIONSHIP, MARRIAGE);
+                templateData.put(MARRIAGE_OR_CIVIL_PARTNERSHIP, MARRIAGE);
+                templateData.put(DIVORCE_OR_END_CIVIL_PARTNERSHIP, OF_THE_DIVORCE);
+
+            } else {
+                templateData.put(DIVORCE_OR_DISSOLUTION, TO_END_A_CIVIL_PARTNERSHIP);
+                templateData.put(MARRIAGE_OR_RELATIONSHIP, RELATIONSHIP);
+                templateData.put(MARRIAGE_OR_CIVIL_PARTNERSHIP, CIVIL_PARTNERSHIP);
+                templateData.put(DIVORCE_OR_END_CIVIL_PARTNERSHIP, TO_END_THE_CIVIL_PARTNERSHIP);
+            }
+
+            templateData.put(IS_SOLE, caseData.getApplicationType().isSole());
+            templateData.put(APPLICANT_1_FIRST_NAME, caseData.getApplicant1().getFirstName());
+            templateData.put(APPLICANT_1_MIDDLE_NAME, caseData.getApplicant1().getMiddleName());
+            templateData.put(APPLICANT_1_LAST_NAME, caseData.getApplicant1().getLastName());
+
+            templateData.put(APPLICANT_2_FIRST_NAME, caseData.getApplicant2().getFirstName());
+            templateData.put(APPLICANT_2_MIDDLE_NAME, caseData.getApplicant2().getMiddleName());
+            templateData.put(APPLICANT_2_LAST_NAME, caseData.getApplicant2().getLastName());
+
+            templateData.put(HAS_FINANCIAL_ORDERS, caseData.getApplicant1().getFinancialOrder().toBoolean());
+            templateData.put(HAS_FINANCIAL_ORDER_APPLICANT_1, caseData.getApplicant1().getFinancialOrder().toBoolean());
+            templateData.put(HAS_FINANCIAL_ORDER_APPLICANT_2, caseData.getApplicant2().getFinancialOrder().toBoolean());
+
+            templateData.put(CCD_CASE_REFERENCE, ccdCaseReference);
+            templateData.put(ISSUE_DATE, createdDate.format(TEMPLATE_DATE_FORMAT));
+            templateData.put(MARRIAGE_DATE,
+                ofNullable(caseData.getApplication().getMarriageDetails().getDate())
```

Comment:
I suppose marriage details could be null in civil partnership

Reply:
marriage details are initialised in case data so they will never be null
```
    @JsonUnwrapped()
    @Builder.Default
    private Application application = new Application();
```